src/main/java/core/basesyntax/FigureDrawing.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDrawing {

```

Comment:
Using word "Figure" here is completly against OOP Abstraction principle. How are you going to use this interface for other classes?